sql/src/main/java/io/crate/planner/consumer/UpdatePlanner.java

```diff
@@ -196,8 +196,12 @@ private static Plan upsertByQuery(UpdateAnalyzedStatement.NestedAnalyzedStatemen
 
         Symbol versionSymbol = null;
         if (whereClause.hasVersions()) {
-            versionSymbol = VersionRewriter.get(whereClause.query());
-            whereClause = new WhereClause(whereClause.query(), whereClause.docKeys().orElse(null), whereClause.partitions());
+            VersionExtractor.Result result = VersionExtractor.extractVersionComparisons(whereClause.query());
+            if (result.versions.size() > 1) {
+                throw new UnsupportedOperationException("Only a single _version comparison may occur in the WHERE clause");
```

Comment:
The previous VersionRewriter had an assertion - so I kept the behaviour in that it causes a failure.